paasta_tools/cli/cmds/rerun.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import datetime
+
+from paasta_tools.cli.cmds.status import get_actual_deployments
+from paasta_tools.cli.cmds.status import get_planned_deployments
+from paasta_tools.cli.cmds.status import list_deployed_clusters
+from paasta_tools.cli.utils import execute_chronos_rerun_on_remote_master
+from paasta_tools.cli.utils import figure_out_service_name
+from paasta_tools.cli.utils import lazy_choices_completer
+from paasta_tools.cli.utils import list_instances
+from paasta_tools.cli.utils import list_services
+from paasta_tools.utils import list_clusters
+from paasta_tools.utils import PaastaColors
+from paasta_tools.utils import SPACER
+
+
+EXECUTION_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+
+def type_execution_date(value):
+    try:
+        parsed = datetime.datetime.strptime(value, EXECUTION_DATE_FORMAT)
+    except ValueError:
+        raise argparse.ArgumentTypeError('must be in the format "%s"' % EXECUTION_DATE_FORMAT)
+    return parsed
+
+
+def add_subparser(subparsers):
+    rerun_parser = subparsers.add_parser(
+        'rerun',
+        help="Re-run a scheduled PaaSTA job",
+        description=(
+            "'paasta rerun' creates a copy of the specified PaaSTA scheduled job and executes it immediately. "
```

Comment:
> You don't have to repeat the command paasta rerun, as it shows up in the help right next to the command already in the help message

We seem to be doing that everywhere though... Should I remove the redundancy from every command, in a separate PR?
```
(py27)gcioffi@dev9-devc:~/git/paasta (paasta_rerun_frontend-issue-323) $ git grep -A1 'description=(' paasta_tools/cli/cmds/*.py | grep '"' | sed 's/^.*py-//;s/^ *//'
"'paasta emergency-restart' is useful in situations where the operator "
"'emergency-scale' is used to scale a PaaSTA service instance by scaling it up or down "
"'emergency-start' scales a PaaSTA service instance up to the configured instance count for a "
"'emergency-stop' stops a Marathon service instance by scaling it down to 0. If the "
"'paasta fsm' is used to generate example soa-configs, which is useful during initial "
"'paasta generate-pipeline' is a Yelp-specific tool to interact with Jenkins "
"'paasta info' gathers information about a service from soa-configs "
"'paasta itest' runs 'make itest' in the root of a service directory. "
"'paasta list' inspects the soa-configs directory and lists all of the "
"'paasta list' inspects all of the PaaSTA services declared in the soa-configs "
"'paasta local-run' is useful for simulating how a PaaSTA service would be "
"'paasta logs' works by streaming PaaSTA-related event messages "
"'paasta mark-for-deployment' uses Git as the control-plane, to "
"'paasta metastatus' is used to get the vital statistics about a PaaaSTA "
"'paasta push-to-registry' is a tool to upload a local docker image "
"'paasta rerun' creates a copy of the specified PaaSTA scheduled job and executes it immediately. "
"'paasta rollback' is a human-friendly tool for marking a particular "
"%ss a PaaSTA service in a graceful way. This uses the Git control plane." % upper
"'paasta status' works by SSH'ing to remote PaaSTA masters and "
```